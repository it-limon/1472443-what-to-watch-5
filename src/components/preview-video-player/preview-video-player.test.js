import React from "react";
import renderer from "react-test-renderer";
import PreviewVideoPlayer from "./preview-video-player";

describe(`Render PreviewVideoPlayer`, () => {

  it(`Render PreviewVideoPlayer`, () => {
    const tree = renderer
      .create(
          <PreviewVideoPlayer
            src={`src.avi`}
            poster={`poster.jpg`}
            isPlaying={true}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
