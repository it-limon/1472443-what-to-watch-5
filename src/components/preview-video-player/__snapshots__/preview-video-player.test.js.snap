// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render PreviewVideoPlayer Render PreviewVideoPlayer 1`] = `
<video
  className="player__video"
  muted={true}
  poster="poster.jpg"
  src="src.avi"
/>
`;
